libs/components/src/modal/modal.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  [ngClass]="width"
+  class="tw-max-h-screen tw-my-4 tw-flex tw-flex-col tw-bg-text-contrast tw-border tw-border-solid tw-border-secondary-300 tw-rounded tw-text-main tw-overflow-hidden"
+>
+  <div
+    class="tw-flex tw-gap-4 tw-p-4 tw-border-0 tw-border-b tw-border-solid tw-border-secondary-300 tw-text-main"
+  >
+    <h2 class="tw-text-lg tw-uppercase tw-flex tw-grow tw-mb-0">
+      <ng-content select="[bit-modal-title]"></ng-content>
+    </h2>
+    <button class="tw-border-0 tw-bg-transparent tw-p-0">
+      <i class="bwi bwi-close tw-text-xs tw-font-bold tw-text-main" aria-hidden="true"></i>
+    </button>
+  </div>
+
+  <div class="tw-pb-8 tw-p-4 tw-w-full tw-relative tw-overflow-y-auto">
```

Comment:
Is Width and relative needed here?

```suggestion
  <div class="tw-pb-8 tw-p-4 tw-overflow-y-auto">
```

Reply:
resolved in https://github.com/bitwarden/clients/pull/3053/commits/938f2362c707729d900ffabad29b2289efe7fd5a